src/app/pages/HomePage/Graph/GraphTooltipMobile/index.tsx

```diff
@@ -163,6 +165,20 @@ const useLayerTooltipMap = (network: Network): Record<Layer, TooltipInfo> => {
           }
         : {}),
     },
+    [Layer.pontusx]: {
```

Comment:
So wait, is this supposed to show in Graph/Homepage or no? Because if no, I wouldn't change any code of Graph, but use some partial(can be new) type of ParaTimes, and use only the ones that Graph actually support. Because for every new ParaTime we need to modify the Graph for no reason.

Reply:
Updated type definition, we no longer have to add unused stuff to the graph code.